types/component/AnonymousChat/BaseChatItem.types.ts

```diff
@@ -34,7 +35,7 @@ export interface ChatItemTargetText extends BaseChatItemProps {
 export interface BaseChatItemComponentProps {
   item?: ChatSchema;
   index?: number;
-  type?: 'ANONYMOUS' | 'SIGNED';
+  type: Anonimity;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Replacing the `type` property with `Anonimity` type instead of `'ANONYMOUS' | 'SIGNED'` is a good move. This will ensure that only valid values can be assigned to the `type` property, improving type safety.
```diff
-  type?: 'ANONYMOUS' | 'SIGNED';
+  type: Anonimity;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->